src/commands/snapshot.ts

```diff
@@ -0,0 +1,110 @@
+import {flags} from '@oclif/command'
+import Constants from '../services/constants'
+import {StaticSnapshotOptions} from '../services/static-snapshot-options'
+import StaticSnapshotService from '../services/static-snapshot-service'
+import logger from '../utils/logger'
+import PercyCommand from './percy-command'
+
+export default class Snapshot extends PercyCommand {
+  static description = 'Snapshot a directory of webpages'
```

Comment:
this feels weird to me.  Maybe 'Snapshot a directory of html files'?   Or 'Snapshot a folder containing a pre-built static website`?

Reply:
I like the latter. Users can technically change the extension of files they want to snapshot